plugins/resources/unixfilesystem/src/irods_is_in_host_list.cpp

```diff
@@ -0,0 +1,44 @@
+#include "irods/private/irods_is_in_host_list.hpp"
+
+#include "irods/irods_error.hpp"
+#include "irods/irods_string_tokenize.hpp"
+
+// boost includes
+#include <boost/algorithm/string/predicate.hpp>
+
+// stl includes
+#include <vector>
+
+namespace irods
+{
+    const std::string HOST_LIST("host_list");
+
+    // returns true if there is no host_list or if the current host is in
+    // the list
+    auto is_host_in_host_list(irods::plugin_property_map& prop_map, const std::string& resource_hostname) -> bool
```

Comment:
The data type of `resource_hostname` can be a `const std::string_view`.

Reply:
I like that empty line.  Micromanage much?

Reply:
Looks horrible but done.

Reply:
done